Current NETCONN branch doesn't build

Hello, 

Thank you for your work on this project. It's amazing! I was able to make previous commit build and run flawlessly. Current version doesn't build for some reason (idf 4.3.4). 

```
[ 99%] Built target __idf_websocket_if
[ 99%] Building C object esp-idf/main/CMakeFiles/__idf_main.dir/main.c.obj
/root/src/snapclient/snapclient/main/main.c: In function 'flac_task':
/root/src/snapclient/snapclient/main/main.c:510:34: error: 'scSet' is a pointer; did you mean to use '->'?
         dsp_setup_flow(500, scSet.sr, scSet.chkDur_ms);
                                  ^
                                  ->
/root/src/snapclient/snapclient/main/main.c:510:44: error: 'scSet' is a pointer; did you mean to use '->'?
         dsp_setup_flow(500, scSet.sr, scSet.chkDur_ms);
                                            ^
                                            ->
/root/src/snapclient/snapclient/main/main.c:511:23: error: 'pcm_chunk_message' undeclared (first use in this function); did you mean 'pcm_chunk_message_t'?
         dsp_processor(pcm_chunk_message.payload, pcm_chunk_message.size,
                       ^~~~~~~~~~~~~~~~~
                       pcm_chunk_message_t
/root/src/snapclient/snapclient/main/main.c:511:23: note: each undeclared identifier is reported only once for each function it appears in
/root/src/snapclient/snapclient/main/main.c: In function 'http_get_task':
/root/src/snapclient/snapclient/main/main.c:1409:47: error: 'pcm_chunk_message' undeclared (first use in this function); did you mean 'pcm_chunk_message_t'?
                                 dsp_processor(pcm_chunk_message.payload,
                                               ^~~~~~~~~~~~~~~~~
                                               pcm_chunk_message_t
/root/src/snapclient/snapclient/main/main.c:780:25: warning: variable 'endTime' set but not used [-Wunused-but-set-variable]
     uint64_t startTime, endTime;
                         ^~~~~~~
/root/src/snapclient/snapclient/main/main.c:780:14: warning: variable 'startTime' set but not used [-Wunused-but-set-variable]
     uint64_t startTime, endTime;
              ^~~~~~~~~
/root/src/snapclient/snapclient/main/main.c:2010:36: warning: this statement may fall through [-Wimplicit-fallthrough=]
                       internalState++;
                       ~~~~~~~~~~~~~^~
/root/src/snapclient/snapclient/main/main.c:2014:21: note: here
                     case 5: {
                     ^~~~
make[2]: *** [esp-idf/main/CMakeFiles/__idf_main.dir/build.make:76: esp-idf/main/CMakeFiles/__idf_main.dir/main.c.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:5572: esp-idf/main/CMakeFiles/__idf_main.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
make failed with exit code 2
```

Sorry if this is something obvious.

--- a/main/main.c
+++ b/main/main.c
@@ -507,8 +507,8 @@ void flac_task(void *pvParameters) {
         }
 
 #if CONFIG_USE_DSP_PROCESSOR
-        dsp_setup_flow(500, scSet.sr, scSet.chkDur_ms);
-        dsp_processor(pcm_chunk_message.payload, pcm_chunk_message.size,
+        dsp_setup_flow(500, scSet->sr, scSet->chkDur_ms);
+        dsp_processor(pcmData->fragment->payload, pcmData->fragment->size,
                       dspFlow);
 #endif
 
@@ -1406,8 +1406,8 @@ static void http_get_task(void *pvParameters) {
 
 #if CONFIG_USE_DSP_PROCESSOR
                                 dsp_setup_flow(500, scSet.sr, scSet.chkDur_ms);
-                                dsp_processor(pcm_chunk_message.payload,
-                                              pcm_chunk_message.size, dspFlow);
+                                dsp_processor(pcmData->fragment->payload,
+                                              pcmData->fragment->size, dspFlow);
 #endif
 
                                 insert_pcm_chunk(pcmData);
@@ -1444,8 +1444,8 @@ static void http_get_task(void *pvParameters) {
 
 #if CONFIG_USE_DSP_PROCESSOR
                               dsp_setup_flow(500, scSet.sr, scSet.chkDur_ms);
-                              dsp_processor(pcm_chunk_message.payload,
-                                            pcm_chunk_message.size, dspFlow);
+                              dsp_processor(pcmData->fragment->payload,
+                                            pcmData->fragment->size, dspFlow);
 #endif
 
                               insert_pcm_chunk(pcmData);

